.github/workflows/bun-test.yml

```diff
@@ -10,7 +10,7 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
-    timeout-minutes: 5
```

Comment:
The timeout issue is for just playwright, right? Do we need to increase it for bun tests?

Reply:
umm yeah, refactored it!